corehq/apps/hqadmin/management/commands/get_download_url.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from django.core.management.base import BaseCommand
+
+import magic
+
+from corehq.apps.hqadmin.utils import get_download_url
+
+
+class Command(BaseCommand):
+    help = "Upload a file to blob storage for subsequent download"
+
+    def add_arguments(self, parser):
+        parser.add_argument('file_path')
+        parser.add_argument('--keep-hours', type=int, default=24,
+                            help="Number of hours to store file. Defaults to 24.")
+
+    def handle(self, file_path, keep_hours, **options):
+        name = os.path.basename(file_path)
+        mime_type = magic.from_file(file_path, mime=True)
+        with open(file_path, 'rb') as file_obj:
+            download_url = get_download_url(file_obj, name, mime_type, keep_hours * 60)
```

Comment:
I'm torn - I'm hoping for the most part there won't be any need to set a timeout at all, given the default of 1 day.  The main use-case I have in mind is downloading the file immediately after running a command, though thinking about it again I wonder if a slightly longer default would be better, like 3 days.  Say you kick off something, it takes a few days, finishing over the weekend, it'd be pretty unfortunate for your artifact to timeout before you get a chance to download it.  Depends whether you use the management command or build the util directly into your one-off command, I guess.

At any rate, I'm a little hesitant to put limits on a developer-facing feature like this, thinking we can trust people to be careful, and I don't want to be overly restrictive, not knowing exactly how this might be used.  For instance, I recently did a domain dump, but couldn't download it to my machine directly, so I let it live on the servers for around a week and a half while we sorted out exactly where to transfer it.

That said, there is probably _some_ number we can probably consider a reasonable maximum, even if that's like 3 months, and I'm not opposed to adding it.  Another idea I considered was making the download link also delete the blob - as a sort of single-use URL.  Though again, that limits its utility, so should probably be parameterized.